TracingAmqpToAppNoParentTest is flaky

The TracingAmqpToAppNoParentTest is flaky and fails on CI almost constantly. 

```
021-06-20T17:57:53.8828335Z [INFO] Running io.smallrye.reactive.messaging.amqp.TracingAmqpToAppNoParentTest
2021-06-20T17:57:53.8830894Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl deployAddressesFromConfiguration
2021-06-20T17:57:53.8832655Z INFO: AMQ221080: Deploying address DLQ supporting [ANYCAST]
2021-06-20T17:57:53.8834530Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl deployQueuesFromListQueueConfiguration
2021-06-20T17:57:53.8843901Z INFO: AMQ221003: Deploying ANYCAST queue DLQ on address DLQ
2021-06-20T17:57:53.8856367Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl deployAddressesFromConfiguration
2021-06-20T17:57:53.8858153Z INFO: AMQ221080: Deploying address ExpiryQueue supporting [ANYCAST]
2021-06-20T17:57:53.8860057Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl deployQueuesFromListQueueConfiguration
2021-06-20T17:57:53.8861950Z INFO: AMQ221003: Deploying ANYCAST queue ExpiryQueue on address ExpiryQueue
2021-06-20T17:57:53.8863814Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl deployQueuesFromListQueueConfiguration
2021-06-20T17:57:53.8865619Z INFO: AMQ221003: Deploying MULTICAST queue people on address people
2021-06-20T17:57:53.8866682Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.logs.AuditLogger getMBeanInfo
2021-06-20T17:57:53.8868782Z INFO: AMQ601019: User anonymous@unknown is getting mbean info on target resource: org.apache.activemq.artemis.core.management.impl.AcceptorControlImpl@5ebbde60 []
2021-06-20T17:57:53.8870896Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.logs.AuditLogger getMBeanInfo
2021-06-20T17:57:53.8872989Z INFO: AMQ601019: User anonymous@unknown is getting mbean info on target resource: org.apache.activemq.artemis.core.management.impl.AcceptorControlImpl@a056b26 []
2021-06-20T17:57:53.8876135Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.core.remoting.impl.netty.NettyAcceptor start
2021-06-20T17:57:53.8882034Z INFO: AMQ221020: Started EPOLL Acceptor at 0.0.0.0:61616 for protocols [CORE,AMQP]
2021-06-20T17:57:53.8889554Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.core.remoting.impl.netty.NettyAcceptor start
2021-06-20T17:57:53.8891662Z INFO: AMQ221020: Started EPOLL Acceptor at 0.0.0.0:5672 for protocols [AMQP]
2021-06-20T17:57:53.8906848Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.core.server.impl.LiveOnlyActivation run
2021-06-20T17:57:53.8908170Z INFO: AMQ221007: Server is now live
2021-06-20T17:57:53.8909653Z Jun 20, 2021 5:57:53 PM org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl internalStart
2021-06-20T17:57:53.8911651Z INFO: AMQ221001: Apache ActiveMQ Artemis Message Broker version 2.17.0 [0.0.0.0, nodeID=08933264-d1f1-11eb-b766-000d3a17fbd2] 
2021-06-20T17:57:54.0179896Z Jun 20, 2021 5:57:54 PM org.jboss.weld.environment.deployment.discovery.DiscoveryStrategyFactory create
2021-06-20T17:57:54.0182954Z INFO: WELD-ENV-000020: Using jandex for bean discovery
2021-06-20T17:57:54.0210096Z Jun 20, 2021 5:57:54 PM org.jboss.weld.xml.BeansXmlValidator error
2021-06-20T17:57:54.0212132Z WARN: WELD-001208: Error when validating null@6 against xsd. cvc-complex-type.4: Attribute 'bean-discovery-mode' must appear on element 'beans'.
2021-06-20T17:57:54.0799744Z Jun 20, 2021 5:57:54 PM org.jboss.weld.bootstrap.WeldStartup startContainer
2021-06-20T17:57:54.0802294Z INFO: WELD-000101: Transactional services not available. Injection of @Inject UserTransaction not available. Transactional observers will be invoked synchronously.
2021-06-20T17:57:54.1001239Z Jun 20, 2021 5:57:54 PM org.jboss.weld.bootstrap.MissingDependenciesRegistry handleResourceLoadingException
2021-06-20T17:57:54.1005583Z INFO: WELD-000119: Not generating any bean definitions from io.smallrye.reactive.messaging.metrics.MetricDecorator because of underlying class loading error: Type org.eclipse.microprofile.metrics.MetricRegistry not found.  If this is unexpected, enable DEBUG logging to see the full error.
2021-06-20T17:57:54.1492225Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.extension.ReactiveMessagingExtension afterDeploymentValidation
2021-06-20T17:57:54.1496006Z INFO: SRMSG00224: Analyzing mediator bean: Managed Bean [class io.smallrye.reactive.messaging.amqp.TracingAmqpToAppNoParentTest$MyAppReceivingData] with qualifiers [@Any @Default]
2021-06-20T17:57:54.1556620Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.connectors.ExecutionHolder <init>
2021-06-20T17:57:54.1560053Z INFO: SRMSG00202: Created new Vertx instance
2021-06-20T17:57:54.1561435Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpClientHelper getClient
2021-06-20T17:57:54.1562808Z INFO: SRMSG16201: AMQP broker configured to 127.0.0.1:5672 for channel stuff
2021-06-20T17:57:54.1611822Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.ConnectionHolder lambda$getOrEstablishConnection$1
2021-06-20T17:57:54.1613719Z INFO: SRMSG16212: Establishing connection with AMQP broker
2021-06-20T17:57:54.1622848Z Jun 20, 2021 5:57:54 PM org.jboss.weld.environment.se.WeldContainer fireContainerInitializedEvent
2021-06-20T17:57:54.1624991Z INFO: WELD-ENV-002003: Weld SE container 065ff214-03a7-4a88-999d-8497efdebe76 initialized
2021-06-20T17:57:54.1753470Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger userSuccesfullyLoggedInAudit
2021-06-20T17:57:54.1755164Z INFO: AMQ601715: User anonymous@127.0.0.1:58478 successfully authorized
2021-06-20T17:57:54.1795606Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger createCoreSession
2021-06-20T17:57:54.1803452Z INFO: AMQ601267: User anonymous@127.0.0.1:58478 is creating a core session on target resource ActiveMQServerImpl::serverUUID=08933264-d1f1-11eb-b766-000d3a17fbd2 [with parameters: [0982264e-d1f1-11eb-b766-000d3a17fbd2, artemis, ****, 102400, org.apache.activemq.artemis.protocol.amqp.broker.ActiveMQProtonRemotingConnection@7932d1a2, false, false, false, true, null, org.apache.activemq.artemis.protocol.amqp.broker.AMQPSessionCallback@72e5fbc4, true, OperationContextImpl [22105934] [minimalStore=9223372036854775807, storeLineUp=0, stored=0, minimalReplicated=9223372036854775807, replicationLineUp=0, replicated=0, paged=0, minimalPage=9223372036854775807, pageLineUp=0, errorCode=-1, errorMessage=null, executorsPending=0, executor=OrderedExecutor(tasks=[])], {}]]
2021-06-20T17:57:54.1817102Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger userSuccesfullyLoggedInAudit
2021-06-20T17:57:54.1818527Z INFO: AMQ601715: User anonymous@127.0.0.1:58478 successfully authorized
2021-06-20T17:57:54.1819909Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger serverSessionCreateAddress
2021-06-20T17:57:54.1822242Z INFO: AMQ601262: User artemis@127.0.0.1:58478 is creating address on target resource: 0982264e-d1f1-11eb-b766-000d3a17fbd2 [with parameters: [Address [name=no-parent-stuff, id=0, routingTypes={MULTICAST}, autoCreated=false, paused=false], true]]
2021-06-20T17:57:54.1824267Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger getMBeanInfo
2021-06-20T17:57:54.1831443Z INFO: AMQ601019: User anonymous@127.0.0.1:58478 is getting mbean info on target resource: org.apache.activemq.artemis.core.management.impl.AddressControlImpl@1a554104 []
2021-06-20T17:57:54.1833536Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$1
2021-06-20T17:57:54.1834707Z INFO: Starting AMQP sender to write 10 messages
2021-06-20T17:57:54.1847864Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger userSuccesfullyLoggedInAudit
2021-06-20T17:57:54.1849111Z INFO: AMQ601715: User anonymous@127.0.0.1:58476 successfully authorized
2021-06-20T17:57:54.1911418Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger coreSendMessage
2021-06-20T17:57:54.1922920Z INFO: AMQ601500: User artemis@unknown is sending a message AMQPStandardMessage( [durable=false, messageID=0, address=no-parent-stuff, size=49, applicationProperties={}, properties=Properties{messageId=null, userId=null, to='no-parent-stuff', subject='null', replyTo='null', correlationId=null, contentType=null, contentEncoding=null, absoluteExpiryTime=null, creationTime=null, groupId='null', groupSequence=null, replyToGroupId='null'}, extraProperties = null], with Context: RoutingContextImpl(Address=null, routingType=null, PreviousAddress=null previousRoute:null, reusable=null, version=0)
2021-06-20T17:57:54.1925580Z ..................................................
2021-06-20T17:57:54.1925830Z 
2021-06-20T17:57:54.1926700Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger coreSendMessage
2021-06-20T17:57:54.1971823Z INFO: AMQ601500: User artemis@unknown is sending a message AMQPStandardMessage( [durable=false, messageID=0, address=no-parent-stuff, size=49, applicationProperties={}, properties=Properties{messageId=null, userId=null, to='no-parent-stuff', subject='null', replyTo='null', correlationId=null, contentType=null, contentEncoding=null, absoluteExpiryTime=null, creationTime=null, groupId='null', groupSequence=null, replyToGroupId='null'}, extraProperties = null], with Context: RoutingContextImpl(Address=no-parent-stuff, routingType=null, PreviousAddress=null previousRoute:null, reusable=null, version=0)
2021-06-20T17:57:54.1976625Z ..................................................
2021-06-20T17:57:54.1978607Z 
2021-06-20T17:57:54.1985888Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger coreSendMessage
2021-06-20T17:57:54.1989929Z INFO: AMQ601500: User artemis@unknown is sending a message AMQPStandardMessage( [durable=false, messageID=0, address=no-parent-stuff, size=49, applicationProperties={}, properties=Properties{messageId=null, userId=null, to='no-parent-stuff', subject='null', replyTo='null', correlationId=null, contentType=null, contentEncoding=null, absoluteExpiryTime=null, creationTime=null, groupId='null', groupSequence=null, replyToGroupId='null'}, extraProperties = null], with Context: RoutingContextImpl(Address=no-parent-stuff, routingType=null, PreviousAddress=null previousRoute:null, reusable=null, version=0)
2021-06-20T17:57:54.1992613Z ..................................................
2021-06-20T17:57:54.1992858Z 
2021-06-20T17:57:54.1993711Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$0
2021-06-20T17:57:54.1995095Z INFO: Producer sent message 0
2021-06-20T17:57:54.1996026Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$0
2021-06-20T17:57:54.1996963Z INFO: Producer sent message 1
2021-06-20T17:57:54.1997912Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$0
2021-06-20T17:57:54.1998827Z INFO: Producer sent message 2
2021-06-20T17:57:54.1999790Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger coreSendMessage
2021-06-20T17:57:54.2003672Z INFO: AMQ601500: User artemis@unknown is sending a message AMQPStandardMessage( [durable=false, messageID=0, address=no-parent-stuff, size=49, applicationProperties={}, properties=Properties{messageId=null, userId=null, to='no-parent-stuff', subject='null', replyTo='null', correlationId=null, contentType=null, contentEncoding=null, absoluteExpiryTime=null, creationTime=null, groupId='null', groupSequence=null, replyToGroupId='null'}, extraProperties = null], with Context: RoutingContextImpl(Address=no-parent-stuff, routingType=null, PreviousAddress=null previousRoute:null, reusable=null, version=0)
2021-06-20T17:57:54.2006535Z ..................................................
2021-06-20T17:57:54.2006778Z 
2021-06-20T17:57:54.2007633Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger coreSendMessage
2021-06-20T17:57:54.2011482Z INFO: AMQ601500: User artemis@unknown is sending a message AMQPStandardMessage( [durable=false, messageID=0, address=no-parent-stuff, size=49, applicationProperties={}, properties=Properties{messageId=null, userId=null, to='no-parent-stuff', subject='null', replyTo='null', correlationId=null, contentType=null, contentEncoding=null, absoluteExpiryTime=null, creationTime=null, groupId='null', groupSequence=null, replyToGroupId='null'}, extraProperties = null], with Context: RoutingContextImpl(Address=no-parent-stuff, routingType=null, PreviousAddress=null previousRoute:null, reusable=null, version=0)
2021-06-20T17:57:54.2040496Z ..................................................
2021-06-20T17:57:54.2040795Z 
2021-06-20T17:57:54.2044778Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger coreSendMessage
2021-06-20T17:57:54.2057679Z INFO: AMQ601500: User artemis@unknown is sending a message AMQPStandardMessage( [durable=false, messageID=0, address=no-parent-stuff, size=49, applicationProperties={}, properties=Properties{messageId=null, userId=null, to='no-parent-stuff', subject='null', replyTo='null', correlationId=null, contentType=null, contentEncoding=null, absoluteExpiryTime=null, creationTime=null, groupId='null', groupSequence=null, replyToGroupId='null'}, extraProperties = null], with Context: RoutingContextImpl(Address=no-parent-stuff, routingType=null, PreviousAddress=null previousRoute:null, reusable=null, version=0)
2021-06-20T17:57:54.2093850Z ..................................................
2021-06-20T17:57:54.2094130Z 
2021-06-20T17:57:54.2097814Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger coreSendMessage
2021-06-20T17:57:54.2110888Z INFO: AMQ601500: User artemis@unknown is sending a message AMQPStandardMessage( [durable=false, messageID=0, address=no-parent-stuff, size=49, applicationProperties={}, properties=Properties{messageId=null, userId=null, to='no-parent-stuff', subject='null', replyTo='null', correlationId=null, contentType=null, contentEncoding=null, absoluteExpiryTime=null, creationTime=null, groupId='null', groupSequence=null, replyToGroupId='null'}, extraProperties = null], with Context: RoutingContextImpl(Address=no-parent-stuff, routingType=null, PreviousAddress=null previousRoute:null, reusable=null, version=0)
2021-06-20T17:57:54.2137584Z ..................................................
2021-06-20T17:57:54.2155019Z 
2021-06-20T17:57:54.2155913Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger coreSendMessage
2021-06-20T17:57:54.2159896Z INFO: AMQ601500: User artemis@unknown is sending a message AMQPStandardMessage( [durable=false, messageID=0, address=no-parent-stuff, size=49, applicationProperties={}, properties=Properties{messageId=null, userId=null, to='no-parent-stuff', subject='null', replyTo='null', correlationId=null, contentType=null, contentEncoding=null, absoluteExpiryTime=null, creationTime=null, groupId='null', groupSequence=null, replyToGroupId='null'}, extraProperties = null], with Context: RoutingContextImpl(Address=no-parent-stuff, routingType=null, PreviousAddress=null previousRoute:null, reusable=null, version=0)
2021-06-20T17:57:54.2162571Z ..................................................
2021-06-20T17:57:54.2162976Z 
2021-06-20T17:57:54.2163847Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger coreSendMessage
2021-06-20T17:57:54.2167717Z INFO: AMQ601500: User artemis@unknown is sending a message AMQPStandardMessage( [durable=false, messageID=0, address=no-parent-stuff, size=49, applicationProperties={}, properties=Properties{messageId=null, userId=null, to='no-parent-stuff', subject='null', replyTo='null', correlationId=null, contentType=null, contentEncoding=null, absoluteExpiryTime=null, creationTime=null, groupId='null', groupSequence=null, replyToGroupId='null'}, extraProperties = null], with Context: RoutingContextImpl(Address=no-parent-stuff, routingType=null, PreviousAddress=null previousRoute:null, reusable=null, version=0)
2021-06-20T17:57:54.2170406Z ..................................................
2021-06-20T17:57:54.2170650Z 
2021-06-20T17:57:54.2171486Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$0
2021-06-20T17:57:54.2172442Z INFO: Producer sent message 3
2021-06-20T17:57:54.2219503Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$0
2021-06-20T17:57:54.2220451Z INFO: Producer sent message 4
2021-06-20T17:57:54.2221395Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$0
2021-06-20T17:57:54.2222327Z INFO: Producer sent message 5
2021-06-20T17:57:54.2223387Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$0
2021-06-20T17:57:54.2224342Z INFO: Producer sent message 6
2021-06-20T17:57:54.2225282Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$0
2021-06-20T17:57:54.2226205Z INFO: Producer sent message 7
2021-06-20T17:57:54.2227155Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$0
2021-06-20T17:57:54.2228084Z INFO: Producer sent message 8
2021-06-20T17:57:54.2229027Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger coreSendMessage
2021-06-20T17:57:54.2233016Z INFO: AMQ601500: User artemis@unknown is sending a message AMQPStandardMessage( [durable=false, messageID=0, address=no-parent-stuff, size=49, applicationProperties={}, properties=Properties{messageId=null, userId=null, to='no-parent-stuff', subject='null', replyTo='null', correlationId=null, contentType=null, contentEncoding=null, absoluteExpiryTime=null, creationTime=null, groupId='null', groupSequence=null, replyToGroupId='null'}, extraProperties = null], with Context: RoutingContextImpl(Address=no-parent-stuff, routingType=null, PreviousAddress=null previousRoute:null, reusable=null, version=0)
2021-06-20T17:57:54.2235861Z ..................................................
2021-06-20T17:57:54.2236112Z 
2021-06-20T17:57:54.2237261Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.ConnectionHolder lambda$getOrEstablishConnection$3
2021-06-20T17:57:54.2238600Z INFO: SRMSG16213: Connection with AMQP broker established
2021-06-20T17:57:54.2239688Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger createCoreSession
2021-06-20T17:57:54.2246405Z INFO: AMQ601267: User anonymous@127.0.0.1:58476 is creating a core session on target resource ActiveMQServerImpl::serverUUID=08933264-d1f1-11eb-b766-000d3a17fbd2 [with parameters: [0984be5f-d1f1-11eb-b766-000d3a17fbd2, artemis, ****, 102400, org.apache.activemq.artemis.protocol.amqp.broker.ActiveMQProtonRemotingConnection@5e95f49f, false, false, false, true, null, org.apache.activemq.artemis.protocol.amqp.broker.AMQPSessionCallback@5b886664, true, OperationContextImpl [652150727] [minimalStore=9223372036854775807, storeLineUp=0, stored=0, minimalReplicated=9223372036854775807, replicationLineUp=0, replicated=0, paged=0, minimalPage=9223372036854775807, pageLineUp=0, errorCode=-1, errorMessage=null, executorsPending=0, executor=OrderedExecutor(tasks=[])], {}]]
2021-06-20T17:57:54.2251783Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger userSuccesfullyLoggedInAudit
2021-06-20T17:57:54.2253099Z INFO: AMQ601715: User anonymous@127.0.0.1:58476 successfully authorized
2021-06-20T17:57:54.2254147Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpUsage lambda$produce$0
2021-06-20T17:57:54.2255084Z INFO: Producer sent message 9
2021-06-20T17:57:54.2256005Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger createQueue
2021-06-20T17:57:54.2260971Z INFO: AMQ601065: User artemis@127.0.0.1:58476 is creating a queue on target resource: ServerSessionImpl() [with parameters: [QueueConfiguration [id=null, name=3f5d4c0f-a1af-4a1d-8fe4-739875f51a03, address=no-parent-stuff, routingType=MULTICAST, filterString=null, durable=false, user=null, maxConsumers=null, exclusive=null, groupRebalance=null, groupRebalancePauseDispatch=null, groupBuckets=null, groupFirstKey=null, lastValue=null, lastValueKey=null, nonDestructive=null, purgeOnNoConsumers=null, enabled=null, consumersBeforeDispatch=null, delayBeforeDispatch=null, consumerPriority=null, autoDelete=null, autoDeleteDelay=null, autoDeleteMessageCount=null, ringSize=null, configurationManaged=null, temporary=true, autoCreateAddress=null, internal=null, transient=null, autoCreated=null]]]
2021-06-20T17:57:54.2265143Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger getMBeanInfo
2021-06-20T17:57:54.2267233Z INFO: AMQ601019: User anonymous@127.0.0.1:58476 is getting mbean info on target resource: org.apache.activemq.artemis.core.management.impl.QueueControlImpl@a0734dc []
2021-06-20T17:57:54.2269306Z Jun 20, 2021 5:57:54 PM org.apache.activemq.artemis.logs.AuditLogger createCoreConsumer
2021-06-20T17:57:54.2271193Z INFO: AMQ601265: User artemis@127.0.0.1:58476 is creating a core consumer on target resource ServerSessionImpl() [with parameters: [0, 3f5d4c0f-a1af-4a1d-8fe4-739875f51a03, null, 0, false, false, null]]
2021-06-20T17:57:54.2272867Z Jun 20, 2021 5:57:54 PM io.smallrye.reactive.messaging.amqp.AmqpConnector getStreamOfMessages
2021-06-20T17:57:54.2274268Z INFO: SRMSG16203: AMQP Receiver listening address no-parent-stuff
2021-06-20T17:58:04.2507567Z [-- Mutiny had to drop the following exception --]
2021-06-20T17:58:04.2510237Z Exception received by: io.smallrye.mutiny.helpers.UniCallbackSubscriber.onFailure(UniCallbackSubscriber.java:57)
2021-06-20T17:58:04.2512383Z io.vertx.core.impl.NoStackTraceThrowable: Connection disconnected
2021-06-20T17:58:04.2513817Z [------------------------------------------------]
2021-06-20T17:58:04.2649617Z Jun 20, 2021 5:58:04 PM org.jboss.weld.environment.se.WeldContainer shutdown
2021-06-20T17:58:04.2664817Z INFO: WELD-ENV-002001: Weld SE container 065ff214-03a7-4a88-999d-8497efdebe76 shut down
2021-06-20T17:58:04.2666794Z Jun 20, 2021 5:58:04 PM org.apache.activemq.artemis.core.server.impl.ServerSessionImpl connectionFailed
2021-06-20T17:58:04.2669030Z WARN: AMQ222061: Client connection failed, clearing up resources for session 0982264e-d1f1-11eb-b766-000d3a17fbd2
2021-06-20T17:58:04.2670869Z Jun 20, 2021 5:58:04 PM org.apache.activemq.artemis.core.server.impl.ServerSessionImpl connectionFailed
2021-06-20T17:58:04.2672813Z WARN: AMQ222107: Cleared up resources for session 0982264e-d1f1-11eb-b766-000d3a17fbd2
2021-06-20T17:58:04.3956727Z Jun 20, 2021 5:58:04 PM org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl stop
2021-06-20T17:58:04.3969946Z INFO: AMQ221002: Apache ActiveMQ Artemis Message Broker version 2.17.0 [08933264-d1f1-11eb-b766-000d3a17fbd2] stopped, uptime 10.645 seconds
2021-06-20T17:58:04.4022373Z [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 10.676 s <<< FAILURE! - in io.smallrye.reactive.messaging.amqp.TracingAmqpToAppNoParentTest
2021-06-20T17:58:04.4029467Z [ERROR] testFromAmqpToAppWithNoParent  Time elapsed: 10.287 s  <<< ERROR!
2021-06-20T17:58:04.4041853Z org.awaitility.core.ConditionTimeoutException: Condition with io.smallrye.reactive.messaging.amqp.TracingAmqpToAppNoParentTest was not fulfilled within 10 seconds.
2021-06-20T17:58:04.4054282Z 	at io.smallrye.reactive.messaging.amqp.TracingAmqpToAppNoParentTest.testFromAmqpToAppWithNoParent(TracingAmqpToAppNoParentTest.java:109)
2021-06-20T17:58:04.4066866Z 
```

--- a/smallrye-reactive-messaging-amqp/src/test/java/io/smallrye/reactive/messaging/amqp/TracingAmqpToAppNoParentTest.java
+++ b/smallrye-reactive-messaging-amqp/src/test/java/io/smallrye/reactive/messaging/amqp/TracingAmqpToAppNoParentTest.java
@@ -100,6 +100,8 @@ public class TracingAmqpToAppNoParentTest extends AmqpBrokerTestBase {
         container = weld.initialize();
         MyAppReceivingData bean = container.getBeanManager().createInstance().select(MyAppReceivingData.class).get();
 
+        await().until(() -> isAmqpConnectorReady(container));
+
         AtomicInteger count = new AtomicInteger();
 
         usage.produce("no-parent-stuff", 10, count::getAndIncrement);

--- a/smallrye-reactive-messaging-amqp/src/test/java/io/smallrye/reactive/messaging/amqp/TracingAmqpToAppToAmqpTest.java
+++ b/smallrye-reactive-messaging-amqp/src/test/java/io/smallrye/reactive/messaging/amqp/TracingAmqpToAppToAmqpTest.java
@@ -118,6 +118,8 @@ public class TracingAmqpToAppToAmqpTest extends AmqpBrokerTestBase {
 
         container = weld.initialize();
 
+        await().until(() -> isAmqpConnectorReady(container));
+
         AtomicInteger count = new AtomicInteger();
         List<SpanContext> producedSpanContexts = new CopyOnWriteArrayList<>();
         usage.produce("parent-topic", 10, () -> AmqpMessage.create()

--- a/smallrye-reactive-messaging-amqp/src/test/java/io/smallrye/reactive/messaging/amqp/TracingAmqpToAppWithParentTest.java
+++ b/smallrye-reactive-messaging-amqp/src/test/java/io/smallrye/reactive/messaging/amqp/TracingAmqpToAppWithParentTest.java
@@ -108,6 +108,8 @@ public class TracingAmqpToAppWithParentTest extends AmqpBrokerTestBase {
         container = weld.initialize();
         MyAppReceivingData bean = container.getBeanManager().createInstance().select(MyAppReceivingData.class).get();
 
+        await().until(() -> isAmqpConnectorReady(container));
+
         AtomicInteger count = new AtomicInteger();
         List<SpanContext> producedSpanContexts = new CopyOnWriteArrayList<>();
 

--- a/smallrye-reactive-messaging-amqp/src/test/java/io/smallrye/reactive/messaging/amqp/TracingAppToAmqpTest.java
+++ b/smallrye-reactive-messaging-amqp/src/test/java/io/smallrye/reactive/messaging/amqp/TracingAppToAmqpTest.java
@@ -106,6 +106,8 @@ public class TracingAppToAmqpTest extends AmqpBrokerTestBase {
 
         container = weld.initialize();
 
+        await().until(() -> isAmqpConnectorReady(container));
+
         await().until(() -> payloads.size() >= 10);
         assertThat(payloads).containsExactly(0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
 

